app/assets/stylesheets/overrides.css.less

```diff
@@ -214,6 +206,77 @@ li.crop-hierarchy {
   margin: 40px 0px 0px 0px !important;
 }
 
+.search-crops {
+  padding: 80px 0 80px 0;
+}
+
+.find-growers {
+  position: relative;
+  margin-top: 20px;
+}
+
+.title-banner {
+  position: relative;
+  background-image: image-url('veggies.jpg');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  min-height: 280px;
+  padding: 30px;
+  margin: 0 -15px 0 -15px;
+
+  @media (max-width: @screen-md-min) {
+    background-size: @screen-md-min;
+  }
+
+  h1 {
+    font-size: 60px;
```

Comment:
Actually, can we use Bootstrap less variables please!

Bootstrap itself is based on a base pixel size, and then has things like 
@headingblah (I forget the exact names) to set the sizes of everything else.

On 27/03/2015 12:09 am, Taylor Griffin wrote:
>
> In app/assets/stylesheets/overrides.css.less 
> <https://github.com/Growstuff/growstuff/pull/697#discussion_r27210396>:
>
> > +
> > +.title-banner {
> > +  position: relative;
> > +  background-image: image-url('veggies.jpg');
> > +  background-repeat: no-repeat;
> > +  background-size: 100%;
> > +  min-height: 280px;
> > +  padding: 30px;
> > +  margin: 0 -15px 0 -15px;
> > +
> > +  @media (max-width: @screen-md-min) {
> > +    background-size: @screen-md-min;
> > +  }
> > +
> > +  h1 {
> > +    font-size: 60px;
>
> Totally. Shouldn't be a problem.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Growstuff/growstuff/pull/697/files#r27210396>.
>

-- 
Alex "Skud" Bayley
skud@growstuff.org
http://growstuff.org/